internal/command/command.go

```diff
@@ -125,6 +145,25 @@ var CmdUpdateRepo = &Command{
 	},
 }
 
+func defaultOutput() (string, error) {
+	const yyyyMMddHHmmss = "20060102150405" // Expected format by time library
+
+	path := filepath.Join(os.TempDir(), "generator-"+time.Now().Format(yyyyMMddHHmmss))
+
+	if _, err := os.Stat(path); err == nil {
```

Comment:
nit: I would recommend writing this like:

```
	_, err := os.Stat(path)
	switch {
	case os.IsNotExist(err):
		if err := os.Mkdir(path, 0755); err != nil {
			return "", fmt.Errorf("unable to create default output path '%s': %w", path, err)
		}
	case err == nil:
		return "", fmt.Errorf("default output path already exists: %s", path)
	default:
		return "", fmt.Errorf("unable to check directory '%s': %w", path, err)
	}
```

Reply:
Done